CHANGELOG.md

```diff
@@ -2,29 +2,27 @@
 
 ## Features
 
+* **css:** Reduce base heading sizes
 * **component:** Add Split component (#326)
 * **component:** New picto component, deprecating the previous picto card (#382)
 * **template:** New multi-column layout container with up to four even columns (#233)
 * **css:** Add a theme class for alt background colors
 * **css:** Add logo and wordmark components (#665)
 * **css:** Add support for mozilla, pocket, and vpn logos to hero and callout components (#663)
 * **assets:** (breaking) Update @mozilla-protocol/assets to 4.0.0
-* **css:** Add Section Heading component (Fix #664)
-* **css:** Add horizontal spacing tokens (#345)
-* **css:** Add vertical spacing tokens (#536)
-
-## Migration Tips
-
-* See notes for [Protocol Assets 4.0.0](https://github.com/mozilla/protocol-assets/blob/main/CHANGELOG.md#migration-tips)
```

Comment:
Lost this poorly placed migration tip in the rebase.

Reply:
That got moved down to the end https://github.com/mozilla/protocol/pull/677#discussion_r608182658
The diff looks funky because I moved the migration tips to after the bug fix.